enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,175 @@
+current cluster to kubernetes 1.24 based OpenShift 4.11, runtime will be migrated from docker to containerd
+
+## Motivation
+
+In Kubernetes, the CRI interface is used to talk to a container runtime. The design of CRI is to be able
+to run a CRI implementation as a separate binary. However currently the CRI of docker (a.k.a. dockershim) 
+is part of kubelet code, runs as part of kubelet and is tightly coupled with kubelet's lifecycle. From kubernetes
+1.24 onwards dockershim will be removed from kubelet code. Currently WMCO uses docker as default run time. Aim is
+to make containerd as default runtime and move away from docker before dokershim has been decoupled from kubelet.
+
+### Goals
+
+As part of this enhancement we plan to do the following:
+* Make containerd as default run time. 
+* When upgrade happens from older release newer one, containerd will become default run time.  
+
+### Non-Goals
+
+* De-configuring docker run time is not part of this enhancement.
+* Refactoring or re-design of WMCO due to dockershim deprecation.
+* Switching between docker and containerd runtimes is not supported.
+
+## Proposal
+
+To make containerd as default run time, containerd should be installed before kueblet and 
+kubelet parameter need to be updated so that containerd will become default run time. In 
+upgrade case each Windows VM configured by previous versions of the WMCO has its Machine 
+object deleted, resulting in the drain and deletion of the Windows node, and the termination
+then followed by recreation of the VM. The upgraded WMCO instance will then be able to configure 
+VMs that will be created with containerd as default runtime. All minimum requirement stated 
+as part of  WMCO upgrade will be applicable here as well.
+
+Containerd Migration plan
+    * Containerd will become default runtime as part of OpenShift 4.11 
+    * For current usage, we plan to introduce feature flag and release it only for community operator.
+    * If it is downgraded same thing will apply. windows VM Will be de-configured and re-configured with
+      the WMCO supported runtime. ( Note : downgrades are not supported by the OLM)
+
+### User Stories
+
+Stories can be found within the [Windows Containers: containerd](https://issues.redhat.com/browse/WINC-505)
+
+### Justification
+
+If we dont make containerd as default runtime, we will be left out with Mirantis supported dockershim
+and will have to re-deisgn WMCO to incorporate Mirantis dockershim. If we want to use CRI-O for windows, 
+the amount of time and engineering efforts involved making CRI-O to work for windows is huge. This 
+doesn't do business justification due to the high cost and time. Containerd is widely adopted and supported 
+by open source community. Also, Microsoft is major contributor in windows containerd development and making 
+containerd as default runtime for their kubernetes offerings. Most of the windows supported k8s orchestrators
+already moved towards containerd.
```

Comment:
This paragraph could be framed in a more concise way, could also be bullet pointed for clarity.